tests/fixtures/major-name-modification.json

```diff
@@ -7,7 +7,8 @@
         "type": "Feature",
         "properties": {
           "name": "Bengaluru",
-          "wikidata": "Q40435"
+          "wikidata": "Q40435",
+          "osm:version": 9
```

Comment:
@bkowshik I added another fixture for that: https://github.com/mapbox/osm-compare/blob/f3cfa0c8e59620546f3d26992a334378049a06d9/tests/fixtures/major-name-modification.json#L98-L122